data/kamailio.cfg

```diff
@@ -537,113 +538,121 @@ route[PARSE_X_RTP_STATS] {
+		jansson_set("integer", "ExpectedPacketsSent", "$var(ES)", "$var(tmp_json)");
+		jansson_set("integer", "ComfortNoisePacketsSent", "$var(SPC)", "$var(tmp_json)");
+		jansson_set("integer", "SilencePacketsSent", "$var(SPS)", "$var(tmp_json)");
+		jansson_set("integer", "QuietPacketsSent", "$var(QS)", "$var(tmp_json)");
+		jansson_set("integer", "AudioPacketsReceived", "$var(PR)", "$var(tmp_json)");
+		jansson_set("integer", "ExpectedPacketsReceived", "$var(ER)", "$var(tmp_json)");
+		jansson_set("integer", "AudioOctetsReceived", "$var(OR)", "$var(tmp_json)");
+		jansson_set("integer", "ReceivePacketsLost", "$var(PLR)", "$var(tmp_json)");
+		jansson_set("integer", "SendPacketsLost", "$var(PLS)", "$var(tmp_json)");
+		jansson_set("integer", "ReceiveMaxBurstLost", "$var(BL)", "$var(tmp_json)");
+		jansson_set("integer", "NumLostSequences", "$var(LS)", "$var(tmp_json)");
+		jansson_set("integer", "ReceiveBurstDuration", "$var(RBDU)", "$var(tmp_json)");
+		jansson_set("integer", "ReceiveBurstDensity", "$var(RBDE)", "$var(tmp_json)");
+		jansson_set("string", "SendBurstDuration", "$var(SBDU)", "$var(tmp_json)");
+		jansson_set("string", "SendBurstDensity", "$var(SBDE)", "$var(tmp_json)");
+		jansson_set("string", "UsedEncoder", "$var(EN)", "$var(tmp_json)");
+		jansson_set("string", "UsedDecoder", "$var(DE)", "$var(tmp_json)");
+		jansson_set("integer", "ReceiveJitter", "$var(JIR)", "$var(tmp_json)");
+		jansson_set("integer", "SendJitter", "$var(JIS)", "$var(tmp_json)");
+		jansson_set("integer", "RttMean", "$var(DLME)", "$var(tmp_json)");
+		jansson_set("integer", "RttMin", "$var(DLMI)", "$var(tmp_json)");
+		jansson_set("integer", "RttMax", "$var(DLMA)", "$var(tmp_json)");
+		jansson_set("string", "LocalIP", "$var(LIP)", "$var(tmp_json)");
+		jansson_set("string", "RemoteIP", "$var(RIP)", "$var(tmp_json)");
+		jansson_set("integer", "Mos", "$var(MOS)", "$var(tmp_json)");
+		jansson_set("integer", "DecoderQueue", "$var(DQ)", "$var(tmp_json)");
+		jansson_set("integer", "DropSilenceSample", "$var(DSS)", "$var(tmp_json)");
+		jansson_set("integer", "DropSamples", "$var(DS)", "$var(tmp_json)");
+		jansson_set("integer", "PLCSamples", "$var(PLCS)", "$var(tmp_json)");
+		jansson_set("integer", "JitterSamples", "$var(JS)", "$var(tmp_json)");
+
+		$var(xrtp_stats) = $var(xrtp_stats) + " {\"index\": {\"_index\":\"homer-"+ $timef(%d.%m.%Y) +"\",\"_type\":\""+$var(type)+"\" }}\n";
+		$var(xrtp_stats) = $var(xrtp_stats) + $var(tmp_json) + "\n";
+
+		#!ifdef TO_GRAYLOG
+		$var(short)= "X-RTP-Stats";
+		$var(full) = "X-RTP-Stats";
+		jansson_set("string", "short_message", ""+$var(short)+"", "$var(tmp_json)");
+		jansson_set("string", "full_message", ""+$var(full)+"", "$var(tmp_json)");
+		$var(graylog_json) = $var(tmp_json);
+		#!endif
+		
+		$var(tmp_json) = 0;
+
+		if($sht(x=>$ci::callid_count) == $null) $sht(x=>$ci::callid_count) = 0;
+		$sht(x=>$ci::callid_count) = $sht(x=>$ci::callid_count) + 1;
+		
+		$var(body) = $var(xrtp_stats);
+		route(SEND_ES_DATA);
```

Comment:
Your welcome i still need to add but will do in another pr

jansson_set("string", "host", "$var(host)", "$var(tmp_json)");
jansson_set("string", "timestamp", "$var(isotime)", "$var(tmp_json)");